dist/listbox/ds4/listbox.css

```diff
@@ -1,14 +1,36 @@
-.listbox {
+.select {
+  height: 40px;
   position: relative;
 }
-span.listbox {
+.select::after {
```

Comment:
Why did you change the existing DS4 implementation of listbox to use `::after`? We should just leave the existing code as is (using span or svg element) so there are no breaking changes until we decide to officially remove it.

Reply:
Oh. I see the listbox.css file is now getting all the new `.select` classname rules. But we should leave the existing listbox.css file in place for backwards compatibility. Let's point the new `select` module to a `select.css` only.

Reply:
Yup. Somehow this got generated and then I deleted the `listbox.less` which built it. So ... it's kinda a stray until I reinstate `listbox.less`.